### DISCUSSION TITLE
Improving the wiki documentation

### DISCUSSION THREAD

**Discussion by sakiss** [joomlatools/joomlatools-pages#554] [General]
The issue references the wiki page: https://github.com/joomlatools/joomlatools-pages/wiki/Configuration

The sample configuration has both syntax and runtime errors.

 1.   DEBUG is undefined. Plz use JDEBUG
 2.   Missing comas at the end of the array

```
//Redirects
    'redirects' => [
        '/path/to/foo' => '/path/to/other/foo'  
        '/path/to/bar' => 'http://example.com/path/to/bar'  
     ]

    //Sites
    'sites' => [
        '[*]' => JPATH_ROOT.'/sites/default'  
     ]

     //Template
     'template'        => 'protostar'
     'template_config' => ['fluidContainer' => 1];
```

**Comment by sakiss** [comment]
Made the changes.
Do note: The changes concern only syntax errors.

There seem to be functional issues as well.
e.g. The set og and twitter metadata do not seem to work.

**Reply by johanjanssens** [reply]
@sakiss Thanks much appreciated. The wiki documentation hasn't been updated in quite a while, spending most of my available time working on the code.

I have updated the example here: https://github.com/joomlatools/joomlatools-pages/wiki/Configuration#example-configuration-settings with the config defaults found here: https://github.com/joomlatools/joomlatools-pages/blob/master/code/site/components/com_pages/config.php#L24

I'm keeping track of the list of releases and documentation that still needs to be amended or updated, the list you can find in the following ticket: https://github.com/joomlatools/joomlatools-pages/issues/10 All releases that have not been checked off have features that still need to be documented. Help is most appreciated.

In all cases the PR's contain full docs already, just a matter of copy/pasting, arranging and cleaning up a bit.
